docs/Encyclopedia/pages/ABSOLUTE_LiftOver.md

```diff
@@ -0,0 +1,30 @@
+# ABSOLUTE LiftOver
+
+## Description ##
+
+ABSOLUTE LiftOver is a copy number variation (CNV) pipeline used in GDC genotyping array harmonization.
+
+## Overview ##
+
+The ABSOLUTE LiftOver workflow in the GDC is a copy number variation (CNV) pipeline used for genotyping array harmonization. It converts genomic coordinates from hg19 to GRCh38, ensuring high-quality, curated CNV data for downstream analysis. This pipeline also provides gene-level copy numbers, along with purity and ploidy measurements.
```

Comment:
It took me forever to verify this was from hg19, great job tracking this information down!